core/validation/initial_validation/validate.py

```diff
@@ -1,163 +1,62 @@
-
-    :param workbook: A dictionary where keys are sheet names and values are pandas
-                     DataFrames representing each sheet in the submission.
-    :param schema: A schema that defines which validations to run.
-    :param auth: A dictionary containing authorisation information.
-    :raises ValidationError: If any of the validation functions catch a validation error.
-    """
-    wrong_input_validation(workbook, schema)
-    conflicting_input_validation(workbook, schema)
-
-    if schema.get("authorisation_checks") and auth:
-        authorisation_validation(workbook, auth, schema)
-
-
-def authorisation_validation(workbook: dict[str, pd.DataFrame], auth: dict, schema: dict):
-    """Performs authorisation validation based on the provided schema.
-
-    An 'auth' dictionary is sent to the /ingest API when the user attempts to submit.
-    This dictionary contains variables against which the submission form is checked
-    to ensure that the user is able to submit the selected information based on their
-    user level permissions, which is handled by the frontend.
-
-    :param workbook: A dictionary where keys are sheet names and values are pandas
-                    DataFrames representing each sheet in the submission.
-    :param auth: A dictionary containing authorisation information.
-    :param schema: The schema containing authorisation checks.
-    :raises ValidationError: If authorisation validation fails.
-    """
-    authorisation_checks = list()
-    for sheet, column, row, expected_values, type in schema["authorisation_checks"]:
-        expected_values = auth[type]
-
-        authorisation_checks.append(
-            Check(sheet=sheet, column=column, row=row, expected_values=expected_values, type=type)
-        )
-
-    failures = check_values(workbook, authorisation_checks, UnauthorisedSubmissionFailure)
-
-    if failures:
-        raise ValidationError(validation_failures=failures)
-
-
-def wrong_input_validation(workbook: dict[str, pd.DataFrame], schema: dict):
-    """Performs wrong input validation based on the provided schema.
-
-    Checks that the input for specified cells in the schema corresponds to the expected input based on the schema.
-
-    :param workbook: A dictionary where keys are sheet names and values are pandas
-                     DataFrames representing each sheet in the submission.
-    :param schema: The schema containing wrong input checks.
-    :raises ValidationError: If wrong input validation fails.
+def initial_validate(workbook: dict[str, pd.DataFrame], schema: list[Check], auth: dict | None):
     """
-    wrong_input_checks = schema["wrong_input_checks"]
+    Executes initial checks based on the provided schema.
 
-    failures = check_values(workbook, wrong_input_checks, WrongInputFailure)
+    The submission form structures of TF Round 1 & Round 2 do not permit initial validation as they have already been
+    transformed.
```

Comment:
I think we can probably omit this context here. It could be included as a comment in the TF Round 1 and 2 `IngestDependencies` instantiation instead